contracts/protocol/facets/DisputeResolverHandlerFacet.sol

```diff
@@ -289,6 +289,8 @@ contract DisputeResolverHandlerFacet is IBosonAccountEvents, ProtocolBase {
                 sender
             );
         }
+
+        require(updateApplied || needsApproval, NO_UPDATE_APPLIED);
```

Comment:
Not sure i understand this line, but otherwise this WIP looks good. Why is this needed, and how come it has been added to the DisputeResolverHandler, is this just so that update methods throw in a consistent way ?

Reply:
If no field has been updated or requested to be updated, there is no reason to execute the transaction. It is better to revert and return at least some gas to the user.

Reply:
Ok, got it.